test/cases/wasm/export-imported-global/module.wat

```diff
@@ -0,0 +1,17 @@
+(module
+  (import "./env.js" "n" (global i32))
+  (import "./env.js" "m" (global $g2 f64))
+  (export "v" (global 0))
+  (global $g i32 (get_global 0))
```

Comment:
I don't understand why this is valid. Doesn't it break WebAssembly validation to have a non-constant initialization expression, per https://webassembly.github.io/spec/core/bikeshed/index.html#globals%E2%91%A2 ?

Reply:
![image](https://user-images.githubusercontent.com/1365881/46651817-7b15ce80-cba1-11e8-8427-1b8e47e7ce5a.png)

Reply:
Oh, right, I'll dig further into figuring out why the test doesn't pass in Node 11.

Reply:
which test? (sorry I'm just catching up here)

Reply:
OK, I was just confused. V8 supports this just fine (it's tested in test/mjsunit/wasm/globals.js TestImportedExported), and now I'm wondering if the issue is actually the logic in this patch to improve handling of re-exported globals is responsible for getting a Number rather than a WebAssembly.Global. I'm wondering, for systems that have WebAssembly.Global, should we just wrap the Number in a Global?